transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/TransformTestWizardPage.java

```diff
@@ -229,7 +229,7 @@ public void widgetSelected(final SelectionEvent event) {
         Dependency dep = new Dependency();
         dep.setGroupId("org.apache.camel"); //$NON-NLS-1$
         dep.setArtifactId("camel-test-blueprint"); //$NON-NLS-1$
-        dep.setVersion(Util.getCamelVersion(project));
+        dep.setVersion(Util.camelVersion(project));
```

Comment:
Regarding both the getXXX and isXXX comments, of course these are the traditional way of naming methods, but for quite a while I've been annoyed with the useless repetition of the "get" prefix, even though the corresponding "set" prefix is more than enough to distinguish between "read" vs. "write" methods.  The whole paradigm of using the "get" prefix was setup initially to allow introspective code like EE containers to determine which methods are read methods for variable-based properties and not derived methods or those that calculate values, but most people, especially outside of EE code, have since broadened it's use to include all "read"-like methods.  I've started to see a few other API's, including one or two used by the transformation code, that have dropped the use of the "get" prefix, so that discovery, I've been happily following suit.  In fact, you can see that Sun/Oracle itself has followed this approach in the JCodeModel library that we use.  I believe it offers less useless noise to the code and allows the names to more immediately and directly reflect what the method's purpose is.